tools/ci.py

```diff
@@ -708,13 +708,25 @@ def pkg_matrix(
     sessions = [
         "install",
     ]
+    # OSs that where never included in 3005
```

Comment:
@s0undt3ch and @ScriptAutomate just wanted to give you a heads up on this change. Wanted to get your review to make sure this looks okay. I guess once the tests run I will be able to tell more definitly if this doesn't break things. But also I'm going to create a follow up PR that cleans up a lot of these OSs, because some of these have been added to 3006 and can do upgrade/downgrade tests for that version. I didn't want to make that change in this PR as ti might result in more test failures.

Reply:
Here's the follow up PR: https://github.com/saltstack/salt/pull/65442